lib/modules/uscore_v3.1.0/bodyheight_sequence.rb

```diff
@@ -541,8 +541,7 @@ def patient_ids
           invalid_bindings.each { |invalid| invalid_binding_resources << "#{invalid[:resource]&.resourceType}/#{invalid[:resource].id}" }
           invalid_binding_messages.concat(invalid_bindings.map { |invalid| invalid_binding_message(invalid, binding_def) })
         end
-        assert invalid_binding_messages.blank?, "#{invalid_binding_messages.count} invalid required binding(s) found in #{invalid_binding_resources.count} resources:" \
-                                                "#{invalid_binding_messages.join('. ')}"
+        assert invalid_binding_messages.blank?, "#{invalid_binding_messages.count} invalid required #{'binding'.pluralize(invalid_binding_messages.count)}" " found in #{invalid_binding_resources.count} #{'resource'.pluralize(invalid_binding_resources.count)}: " "#{invalid_binding_messages.join('. ')}"
```

Comment:
It doesn't look like these are being generated in the intended way. Maybe the template needs `\\` instead of just `\`?

Reply:
Yeah, that does appear to be what we were using before. Thanks for the catch; I'll re-generate.